lib/components/CircuitJsonPreview/CircuitJsonPreview.tsx

```diff
@@ -46,6 +45,11 @@ import { capitalizeFirstLetters } from "lib/utils"
 import type { PreviewContentProps, TabId } from "./PreviewContentProps"
 import { version } from "../../../package.json"
 
+interface CircuitJsonErrors {
+  error: string
+  errorMessage: string
+}
```

Comment:
Remove this type. If you are building an array of circuit json errors, just use native circuit json types. There’s no need for an intermediate type and the name is confusing when there are standard error types